config/routes.rb

```diff
@@ -3,4 +3,15 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :users, only:[:index, :show] do
+    resources :posts, only:[:index, :show]
+  end
+
+  # get '/users', to: 'users#index'
+  # get '/users/:id', to: 'users#show'
+
+  # get '/users/:id/posts', to: 'posts#show'
+  # get '/users/:id/posts/:id', to: 'posts#show'
```

Comment:
Kindly remove commented code in your code base as it makes your code look more professional